src/PrestaShopBundle/Resources/config/services/core/circuit_breaker.yml

```diff
@@ -3,11 +3,18 @@ services:
   _defaults:
     public: true
 
-  prestashop.core.circuit_breaker.doctrine_cache:
```

Comment:
Service not anymore available

Reply:
It is, [just below](https://github.com/PrestaShop/PrestaShop/pull/25052/files#diff-5a786fa5a57e5490a73db3e5b6bd561ad20f26af29cc2d1477ec57b7854fa804R6)